test/crowdsale/PausableCrowdsale.test.js

```diff
@@ -0,0 +1,155 @@
+const { assertRevert } = require('../helpers/assertRevert');
```

Comment:
We've redesigned `assertRevert` into a new `shouldFail` helper. Make sure to change that as well!

https://github.com/OpenZeppelin/openzeppelin-solidity/blob/b7d56d54714e01ad023429712eedb496361186bb/test/helpers/test/shouldFail.test.js#L48